FAQ.md

```diff
@@ -0,0 +1,22 @@
+# Foire Aux Questions
```

Comment:
Only because our python tutorials are in french and this was recently collected with and for french users. For now, I think that if we move it to doc, it needs to be translated to english but if we keep it in python tutorials, french it might be more consistent to keep it in french.